### PR TITLE
repos: add release uplift repos (Bug 1806141)

### PR DESCRIPTION
**Pull request by cgsheeh**

### CODE REVIEW

**Comment by cgsheeh** [comment]
This is blocked on the repos being added to Phab, but getting a review so I can land when needed would be great. :+1:

**Review by zzzeid — APPROVED** [review]
This will likely cause a bit of a delay/log jam after deployment since the landing worker has to clone all three repos from scratch.

**Pull request merged**
